src/sql/src/statements/tql.rs

```diff
@@ -55,6 +124,41 @@ pub struct TqlAnalyze {
     pub is_verbose: bool,
 }
 
+impl TqlAnalyze {
+    #[inline]
+    fn format_lookback(&self) -> String {
+        if let Some(lookback) = &self.lookback {
+            format!(", {}\n", lookback)
+        } else {
+            String::default()
+        }
+    }
+
+    #[inline]
+    fn format_is_verbose(&self) -> &str {
+        if self.is_verbose {
+            "VERBOSE"
+        } else {
+            ""
+        }
+    }
+}
+
+impl Display for TqlAnalyze {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let start = &self.start;
+        let end = &self.end;
+        let step = &self.step;
+        let lookback = self.format_lookback();
+        let query = &self.query;
+        let is_verbose = self.format_is_verbose();
+        write!(
+            f,
+            "TQL ANALYZE {is_verbose} ({start}, {end}, {step}{lookback}) {query}"
+        )
+    }
+}
```

Comment:
Seems can still have the extra whitespace issue. Let's inline the code directly.